addon/package.json

```diff
@@ -16,7 +16,7 @@
       "title": "backend URL",
       "description": "URL of the backend service to store pages",
       "type": "string",
-      "value": "https://pageshot.dev.mozaws.net"
+      "value": "https://pageshotfzzzy.dev.mozaws.net/"
```

Comment:
I notice there's a trailing `/` here – that could lead to double slashes.  Really addon/lib/main.js:getBackend() should normalize this (maybe doing `.replace(/\/+$/, "")`, but it doesn't right now.  Of course anyone can enter anything in to the pref, so relying on the pref not having a trailing slash isn't really right either.

I'm guessing we do normalize in some other places, but maybe not consistently.